config/optional/password_policy.settings.yml

```diff
@@ -0,0 +1 @@
+cron_threshold: 250
```

Comment:
This is the same config like provided from the module. There is no need for us to ship this file